### PR TITLE
Fix reference to dependencyinjection to 

### PR DESCRIPTION
**Pull request by mikepizzo**

### CODE REVIEW

**Review by ralfhandl — APPROVED** [review]

**Comment by ralfhandl** [comment]
I get a few warnings:

~~~
[warn]: OmniSharp.MSBuild.ProjectLoader
        Microsoft.Extensions.DependencyInjection.Abstractions 7.0.0 doesn't support netcoreapp3.1 and has not been tested with it. Consider upgrading your TargetFramework to net6.0 or later. You may also set <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.
[warn]: OmniSharp.MSBuild.ProjectLoader
        Microsoft.Extensions.DependencyInjection 7.0.0 doesn't support netcoreapp3.1 and has not been tested with it. Consider upgrading your TargetFramework to net6.0 or later. You may also set <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings> in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.
[warn]: OmniSharp.MSBuild.ProjectLoader
        JetsonsReadWrite depends on Microsoft.Extensions.DependencyInjection (>= 6.1.0) but Microsoft.Extensions.DependencyInjection 6.1.0 was not found. An approximate best match of Microsoft.Extensions.DependencyInjection 7.0.0 was resolved.
~~~

**Review by Hubert-Heijkers — APPROVED** [review]

**Comment by ralfhandl** [comment]
I think the files in folder `.vs` should not be checked in, and `.vs` should be added to `.gitignore`

**Review by ralfhandl — CHANGES_REQUESTED** [review]
Git-ignore folder `.vs`

File: samples/Jetsons/JetsonsReadWrite/JetsonsReadWrite.csproj.user [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ActiveDebugProfile>IIS Express</ActiveDebugProfile>
+  </PropertyGroup>
+</Project>
```
Comment (ralfhandl) [inline samples/Jetsons/JetsonsReadWrite/JetsonsReadWrite.csproj.user:6]: Sounds like user-specific settings: should these really be added to the repo?

**Comment by mikepizzo** [comment]
> I think the files in folder `.vs` should not be checked in, and `.vs` should be added to `.gitignore`, they are probably config/settings for Visual Studio Classic

Yes; these were accidentally brought in when I tried to resolve conflicts in updating the .net framework.  They have been fixed

**Comment by ralfhandl** [comment]
Found two more occurrences of `netcoreapp3.1`, changed them to `net7.0`:
- samples/Jetsons/.vscode/launch.json
- samples/Jetsons/Microsoft.Restier.Providers.InMemory/Microsoft.Restier.Providers.InMemory.csproj

After deleting all `bin` and `obj` folders and rebuilding the Jetsons app launches and works as expected.

**Review by ralfhandl — APPROVED** [review]

**Comment by ralfhandl** [comment]
Still haven't figured out how to launch JetsonsReadWrite 😦

**Comment by mikepizzo** [comment]
> Found two more occurrences of `netcoreapp3.1`, changed them to `net7.0`:
> 
> * samples/Jetsons/.vscode/launch.json
> * samples/Jetsons/Microsoft.Restier.Providers.InMemory/Microsoft.Restier.Providers.InMemory.csproj
> 
> After deleting all `bin` and `obj` folders and rebuilding the Jetsons app launches and works as expected.

Good catch -- thanks!

**Pull request merged**